fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/HeaderNestingJsonBuilderStrategy.java

```diff
@@ -30,15 +30,20 @@ public HeaderNestingJsonBuilderStrategy(Function<Response, JsonNode> baseStrateg
 
     @Override
     public JsonNode apply(Response response) {
+        MappingJsonFactory mappingJsonFactory = new MappingJsonFactory();
         ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
         objectNode.set("response", baseStrategy.apply(response));
```

Comment:
Beautiful! I assume the static string you mentioned above is not needed anymore then